fastlane/Fastfile

```diff
@@ -39,5 +40,33 @@ platform :ios do
       binary_basename: ENV['REM_FL_TESTS_SLATHER_BASENAME'],
       ignore: '*.{h,m}')
   end
+
+  lane :build_swift_package do
+    xcodebuild(
+      clean: true,
+      build: true,
+      scheme: 'RSDKUtilsMain',
+      destination: 'platform=iOS Simulator,name=' + ENV['REM_FL_TESTS_DEVICE'] || 'iPhone 11',
+      xcargs: '"APPLICATION_EXTENSION_API_ONLY=YES"'
+    )
+    xcodebuild(
+      clean: true,
+      build: true,
+      scheme: 'RSDKUtilsMain',
+      destination: 'platform=watchOS Simulator,name=Apple Watch Series 6 - 40mm'
+    )
+    xcodebuild(
+      clean: true,
+      build: true,
+      scheme: 'RSDKUtilsNimble',
+      destination: 'platform=iOS Simulator,name=' + ENV['REM_FL_TESTS_DEVICE'] || 'iPhone 11'
+    )
+    xcodebuild(
+      clean: true,
+      build: true,
+      scheme: 'RSDKUtilsTestHelpers',
+      destination: 'platform=iOS Simulator,name=' + ENV['REM_FL_TESTS_DEVICE'] || 'iPhone 11'
+    )
+  end
```

Comment:
Can the duplication here be reduced with a function or a loop?

Reply:
Yeah. good idea